print-ldp.c

```diff
@@ -372,6 +372,23 @@ ldp_tlv_print(netdissect_options *ndo,
         }
 	break;
 
+    case LDP_TLV_HOP_COUNT:
+        TLV_TCHECK(1);
+	ND_PRINT("\n\t      Hop Count: %u", GET_U_1(tptr));
```

Comment:
Indentation is off in the new code.

Reply:
There seems to be a wild mix of tabs and whitespaces in the current code. - Just check sed -i 's/^        /\t/' print-ldp.c; git diff
What are the new house rules ? Tab or whitespace or just whitespace for indentation ?

Reply:
> Tab or whitespace or just whitespace for indentation ?

Currently there are:
```
$ git grep '^   ' '*.c' |wc -l
46703        (TABS)
$ git grep '^        ' '*.c' |wc -l
14657        (8 SPACES)
$ git grep '^    [^ ]' '*.c' |wc -l
7738         (4 SPACES)
```
For new sources, "tab" should be used.

The rule in CONTRIBUTING.md is "Use the style of the modified sources."
```
For print-ldp.c:
$ git grep '^   ' print-ldp.c |wc -l
118        (TABS)
$ git grep '^        ' print-ldp.c |wc -l
200        (8 SPACES)
$ git grep '^    [^ ]' print-ldp.c |wc -l
136        (4 SPACES)
```
For this one there is no obvious style. We have to re-format the indentation and therefore choose between tabs, 8 spaces and 4 spaces.

Reply:
Yes, it is quite inconsistent as it is now. If possible, try to keep it consistent within the same case block at least.

Reply:
This should be done before this PR as a separate commit.

tabs ?

Reply:
I could redo it nicely using tabs that are one level of indentation rather than n spaces, but since this problem isn't new, I would also be happy to do it much later, and for now just not make it worse than it is.

Reply:
Or perhaps '4 spaces' is less change. i.e:
1) s/^\t\t/<16 spaces>/
2) s/^\t/<8 spaces>/

Reply:
IMO keeping the git blame history is much more important than a uniform style. - It would help if we leave editor indentation configs at the end of the file.

```/*
 * Local Variables:
 * c-basic-offset: 4
 * indent-tabs-mode: t
 * End:
 */```

Reply:
I have fixed the patch and replaced leading blocks of 8 whitespaces with a tab.